cmd/jaeger/internal/integration/integration.go

```diff
@@ -46,13 +52,20 @@ func (s *StorageIntegration) newDataReceiver(t *testing.T, factories otelcol.Fac
 	cfg, err := configProvider.Get(context.Background(), factories)
 	require.NoError(t, err)
 
+	tel, err := telemetry.New(context.Background(), telemetry.Settings{}, cfg.Service.Telemetry)
```

Comment:
Maybe I'm missing something. You're passing logger to the function, but not using it here. Then in l65 you're reading logger from tel - how is it initialized there?

And what does cfg.Service.Telemetry do? There's no config file changes in this PR.

Reply:
Oops.. sorry, I've caused some confusion here. Previously, I tried to initialize the `zap.NewDevelopment()` and pass it to the data receiver but then I realized that it would be better if I used the similar logic with the OTEL collector by retrieving the log settings from the config file ([open-telemetry/opentelemetry-collector/otelcol/collector.go#L187](https://github.com/open-telemetry/opentelemetry-collector/blob/15201f1e5967fd1bc1e51c118178ab109170fe07/otelcol/collector.go#L187) -> [open-telemetry/opentelemetry-collector/service/service.go#L90](https://github.com/open-telemetry/opentelemetry-collector/blob/15201f1e5967fd1bc1e51c118178ab109170fe07/service/service.go#L90)). I'll delete the logger passed to the function.

`cfg.Service.Telemetry` is the struct that contains telemetry settings, I don't change the config file because OTEL already has the default values for this ([open-telemetry/opentelemetry-collector/otelcol/unmarshaler.go#L45](https://github.com/open-telemetry/opentelemetry-collector/blob/main/otelcol/unmarshaler.go#L45)).

My intention by implementing this `telemetry.New()` is to have the exact logger settings with the OTEL collector instead of initializing a new `zap.NewDevelopment()`